test/run_test.py

```diff
@@ -1296,6 +1302,10 @@ def can_run_in_pytest(test):
 
 
 def get_selected_tests(options) -> List[str]:
+    if IS_ARM64 and IS_LINUX:
+        selected_tests = ARM64_LINUX_TEST_LIST
+        return selected_tests
+
```

Comment:
This will render `run_test.py` unusable with any options one might run, and can hinder say CUDA-on-ARM bringup

If you want to limit testing to just test_modules.py, just modify test.sh